src/plugins/ui_feature/find_and_replace.ts

```diff
@@ -289,5 +386,16 @@ export class FindAndReplacePlugin extends UIPlugin {
         }
       }
     }
+    if (this.searchOptions.specificRange && this.searchOptions.specificRange.sheetId === sheetId) {
```

Comment:
retrospectively, I understand what you have tried to achieve, having the zone still visible even when the highlight is not focues. if so, I think it's worth checking the functional behaviour with @Francois-frgi   but at the very least, the zone should be the same shade of color as the match *, not orange with purple match

* maybe some darker purple, or even the same purple but without transpalrency might do the trick

Reply:
@jash-odoo I like the fact that we can see the range while looking for reference but, for consistency, it might indeed look better to align colors (either putting all in orange or all in purple).

cc: @rrahir